cosmetics-web/spec/features/responsible_persons/create_rp_user_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+RSpec.describe "Resposible person creation journey", type: :feature do
+  let(:responsible_person) { create(:responsible_person_with_user, :with_a_contact_person) }
+
+  before do
+    sign_in as_user: create(:user, first_login: true)
+    stub_notify_mailer
+  end
+
+   scenario "Given as a new user, I should be able to create responsible user as a individual sole trader" do
```

Comment:
This reads a bit awkwardly. How about:

```suggestion
   scenario "creating a responsible user as a individual sole trader" do
```